src/containers/profile/index.jsx

```diff
@@ -76,39 +77,35 @@ export const Profile = () => {
           onClick={() => setMenu({ marginLeft: '-105%' })}>
           <FaWindowClose />
         </button>
-      </section>
+      </div>
 
-      <section className={menu}>
-        <section className={user}>
+      <div className={menu}>
+        <div className={user}>
           <div className={photo} />
           <div className={name}>John Doe</div>
           <div className={role}>student</div>
-        </section>
+        </div>
         <button className={getStatus === 0? 'profile_button_active profile_button': 'profile_button'} onClick={() => toggleButton(0)}>
```

Comment:
you should simplify `className={getStatus === 0? 'profile_button_active profile_button': 'profile_button'}`
className={`${profile_button} ${getStatus === 0 ? 'profile_button_active' : ''}`}

Reply:
Please, use Prettier for formating code